src/index.js

```diff
@@ -161,6 +161,16 @@ function getAdditionalConfig (filename) {
     });
 }
 
+function getCapabilitiesOverrides (filename) {
+    if (fs.existsSync(filename)) {
+        const data = fs.readFileSync(filename, 'utf8');
```

Comment:
I would prefer async call to the `readFile` method, since the `openBrowser` method which uses `_generateCapabilities` function is already `async`.